cmd/utils/config.go

```diff
@@ -604,6 +604,18 @@ func (kCfg *KaiaConfig) SetKaiaConfig(ctx *cli.Context, stack *node.Node) {
 	cfg.TriesInMemory = ctx.Uint64(TriesInMemoryFlag.Name)
 	cfg.LivePruning = ctx.Bool(LivePruningFlag.Name)
 	cfg.LivePruningRetention = ctx.Uint64(LivePruningRetentionFlag.Name)
+	cfg.TxPruning = ctx.Bool(TxPruningFlag.Name)
+	if cfg.TxPruning {
+		cfg.TxPruningRetention = ctx.Uint64(TxPruningRetentionFlag.Name)
+	} else {
+		cfg.TxPruningRetention = 0
+	}
+	cfg.ReceiptPruning = ctx.Bool(ReceiptPruningFlag.Name)
+	if cfg.ReceiptPruning {
+		cfg.ReceiptPruningRetention = ctx.Uint64(ReceiptPruningRetentionFlag.Name)
+	} else {
+		cfg.ReceiptPruningRetention = 0
+	}
```

Comment:
[About value constraint]

- Even if both retention values differ, I guess no issue will be found. Do you think another point of view about it?
- Is it okay if both values are lower than the `LivePruningRetention` value?
- A plan of upper and lower bound?

Reply:
- I think they can differ. At least I know an extreme case where deleting all receipts and block syncs well. In pruning terms, TxPruningRetention = infinity and ReceiptPruningRetention = 0.
- Maybe? The Retention value only affects the [pruneTrieNodeLoop](https://github.com/kaiachain/kaia/blob/35e43b859b658732323d3f67264c46a1475016a0/blockchain/blockchain.go#L1509) and it is unrelated to txs or receipts in the database.
- No need for the upper bound. You can set 9999999999 to signify infinity. However, a lower bound could be a good safety measure. e.g. any arbitrary number like 128.

@2dvorak Can we have the lower bound for all 3 (tx, receipt, state live pruning) retentions?

Reply:
@2dvorak @hyunsooda  I think separate lower-bound PR is better. So I can mention it in the Release note.

Reply:
@hyunsooda @blukat29 OK, I'll create another PR for lower-bounds. Thanks!

Reply:
@hyunsooda PTAL